persistence/repository_test.go

```diff
@@ -1,30 +0,0 @@
-package persistence
```

Comment:
Should these tests be restored?

Reply:
Yeah, like rewritten. Will take care of ASAP

Reply:
I may be misunderstanding the point of this test, but is it redundant to the `test/integration_test`?

Reply:
No, integration test is from the perspective of the whole system.  This is testing that the `db` (nee `persistence`) module is working correctly.

Reply:
Done.